EG23.tex

```diff
@@ -355,6 +380,11 @@ \subsection{Fuel Fabrication}
 %% this?
```

Comment:
Above the table I add some explanation of the source of those informations, which might not correspond to the EG01-23 transition specification.